sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchIndexingBufferedSender.java

```diff
@@ -32,12 +32,14 @@
     }
 
     /**
-     * Gets the batch size.
+     * Gets the number of documents required in a batch for it to be flushed.
+     * <p>
+     * This configuration is only taken into account if auto flushing is enabled.
      *
-     * @return The batch size.
+     * @return The number of documents required before a flush is triggered.
      */
-    public int getBatchSize() {
-        return client.getBatchSize();
+    public int getInitialBatchActionCount() {
```

Comment:
I hide BatchSize in Python for this release. Do we want to expose InitialBatchActionCount to users?

Reply:
I can hide this as well, it doesn't have a major value add right now